src/pages/_app.tsx

```diff
@@ -134,6 +134,11 @@ export const WebwalletContext = createContext<{
 	exportWebwallet: () => string
 		} | null>(null)
 
+export const SearchContext = createContext<{
```

Comment:
Call it "DAOSearchContext"

Reply:
Named it as a generic as this can be used in other places as well, right?